WorkloadManagementSystem/Service/JobManagerHandler.py

```diff
@@ -87,7 +88,7 @@ def initialize(self):
     self.owner = credDict['username']
     self.peerUsesLimitedProxy = credDict['isLimitedProxy']
     self.diracSetup = self.serviceInfoDict['clientSetup']
-    self.maxParametricJobs = self.srv_getCSOption('MaxParametricJobs', MAX_PARAMETRIC_JOBS)
+    self.maxParametricJobs = Operations().getValue('JobScheduling/MaxParametricJobs', MAX_PARAMETRIC_JOBS)
```

Comment:
Same here, does it need the VO param ?

Reply:
I think here it is not.
However, I am not sure that the whole idea of defining MaxParametricJobs per VO is correct. There are two things: the limit for bulk submission defined by the JobManager service for ALL the users and VOs and the limit for bulk submission in the transformations. The former is unique for everybody and checked by the JobManager service. It is static and is not "Operational".  The second one has nothing to do with the JobManager and is more related to the properties of the given transformation than the Operational parameter of the given VO. Just that this Transformation property value must not exceed the common hard limit defined for the JobManager service.

Reply:
I agree with you and I can simply revert this line. And then I would also need to remove the line 202 of TaskManagerAgentBase (which was added here).